src/bot/rules.service.ts

```diff
@@ -0,0 +1,73 @@
+import { EmbedBuilder } from 'discord.js';
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@mikro-orm/nestjs';
+import { UseRequestContext, MikroORM } from '@mikro-orm/core';
+import { EntityRepository } from '@mikro-orm/postgresql';
+import { stripIndent } from 'common-tags';
+import * as cronstrue from 'cronstrue';
+
+import { GuildEntity } from 'src/entities/guild/guild.entity';
+
+@Injectable()
+export class RulesService {
+  constructor(
+    private readonly orm: MikroORM,
+    @InjectRepository(GuildEntity)
+    private readonly guilds: EntityRepository<GuildEntity>,
+  ) {}
+
+  @UseRequestContext()
```

Comment:
Don't think we need to talk to the DB here?

Reply:
Addressed.